tools/buildInstaller.js

```diff
@@ -53,32 +84,46 @@ if (isDarwin) {
   var cert = process.env.CERT || '../brave-authenticode.pfx'
   var certPassword = process.env.CERT_PASSWORD
   if (!certPassword) {
-    throw new Error('Certificate password required. Set environment variable CERT_PASSWORD.')
+    raiseError('Certificate password required. Set environment variable CERT_PASSWORD.')
   }
 
-  // Because both x64 and ia32 creates a RELEASES and a .nupkg file we
-  // need to store the output files in separate directories
-  outDir = path.join(outDir, arch)
+  // sign for widevine
+  const wvExe = buildDir + '/Brave.exe'
+  const wvPlugin = buildDir + '/WidevineCdm/_platform_specific/' + widevineCdmArch + '/widevinecdmadapter.dll'
```

Comment:
we will also need `mac_x64` for `widevinecdmadapter.plugin` on MacOS
![image](https://user-images.githubusercontent.com/11330831/29896576-656f1bf2-8d92-11e7-8ac9-d4dd76569aaa.png)

Reply:
macOS is hardcoded as `mac_x64`:
https://github.com/brave/browser-laptop/pull/10725/files#diff-bbf971d3d6ce3c0f91bf010517f5211aR55

Reply:
ah, my bad